python/lsst/daf/butler/registry/connectionString.py

```diff
@@ -91,13 +93,15 @@ def fromConfig(cls, registryConfig):
             dbAuth = DbAuth(DB_AUTH_PATH, DB_AUTH_ENVVAR)
             auth = dbAuth.getAuth(conStr.drivername, conStr.username, conStr.host,
                                   conStr.port, conStr.database)
-        except DbAuthPermissionsError:
-            # re-raise permission error for safety
-            raise
-        except DbAuthError:
-            # credentials file doesn't exist or no match was found
+        except DbAuthNotFoundError:
+            # credentials file doesn't exist or no matches were found
             pass
+        except Exception:
+            # re-raise everything else
+            raise
```

Comment:
Is this necessary?  Leaving it out should cause the exception to propagate normally.